Kino/MainScreen/View/MyCollectionViewCell~.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MyCollectionViewCell.swift
+//  Kino
+//
+//  Created by Viktoriya Polozova on 19/08/2024.
+//
+
+import UIKit
+
+class MyCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet var imageView: UIImageView!
+    @IBOutlet weak var star: UIImageView!
+    @IBOutlet weak var movieTitle: UILabel!
+    @IBOutlet weak var rating: UILabel!
+    
+    static let identifier = "MyCollectionViewCell"
```

Comment:
you can add it to UICollectionViewCell extension

Reply:
done